### PR TITLE
Saved 6 bytes by using popptr1.

### PR DESCRIPTION
**Pull request by IrgendwerA8**
Example for an popptr1 adaptation. Please verify, as I'm not part of CBM community.

### CODE REVIEW

File: libsrc/cbm/rwcommon.s [hunk 1]
```diff
@@ -22,18 +22,15 @@
...
-        jsr     popax           ; Get buf
-        sta     ptr2
-        stx     ptr2+1
+        sta     ptr2+1          ; Remember -count-1
 
-        lda     #$00
-        sta     ptr3
-        sta     ptr3+1          ; Clear ptr3
+        jsr     popptr1         ; Get buf
+                                ; Y=0 by popptr1
```
Comment (oliverschmidt) [inline libsrc/cbm/rwcommon.s]: Please put this comment one the line above (phrasing it one way or another).

**Comment by IrgendwerA8** [comment]
Fixed.
I've already adapted some more files, what do you prefer, a single pull request with many changes later on or a few smaller ones depending on what I have currently done?

**Comment by oliverschmidt** [comment]
> I've already adapted some more files, what do you prefer, a single pull request with many changes later on or a few smaller ones depending on what I have currently done?

I strongly prefer more smaller pull requests. Especially when it comes to several targets. And even more if you are not able/willing to test the changes on a certain target.

**Comment by oliverschmidt** [comment]
> Please verify, as I'm not part of CBM community.

Is there someone to volunteer?

**Comment by greg-king5** [comment]
I tested this PR.  It looks good, and works perfectly.

**Comment by oliverschmidt** [comment]
@greg-king5: Thanks!

**Pull request merged**